e2e_tests/tests/conftest.py

```diff
@@ -206,3 +208,64 @@ def test_start_timer(request: _pytest.fixtures.SubRequest) -> Iterator[None]:
         # This ends up concatenated to the line pytest prints containing the test file and name.
         print("starting at", time.strftime("%Y-%m-%d %H:%M:%S"))
     yield
+
+
+@pytest.fixture(scope="session")
+def is_multirm_cluster() -> bool:
+    sess = api_utils.admin_session()
+    multi_rm = True
+    try:
+        multi_rm = detproc.check_json(sess, ["det", "master", "config", "show", "--json"])[
+            "additional_resource_managers"
+        ]
+    except KeyError:
+        multi_rm = False
+    return multi_rm
+
+
+@pytest.fixture(scope="session")
+def namespaces_created(is_multirm_cluster: bool) -> Tuple[str, str]:
+    defaultrm_namespace = uuid.uuid4().hex[:8]
+    additionalrm_namespace = uuid.uuid4().hex[:8]
+
+    # Create a namespace in Kubernetes in the each resource manager's Kubernetes cluster.
```

Comment:
Looks like a partial reword was missed in that comment. :)
```suggestion
    # Create a namespace in each resource manager's Kubernetes cluster.
```

Reply:
We have a [ticket](https://hpe-aiatscale.atlassian.net/browse/DET-10437) up for improvements (such as this) that should be made